timemachine/fe/rbfe.py

```diff
@@ -357,8 +360,11 @@ def optimize_coordinates(initial_states, min_cutoff=0.7) -> List[NDArray]:
     for state, coords in zip(initial_states, all_xs):
         displacement_distances = jax_utils.distance_on_pairs(state.x0, coords, box=state.box0)
```

Comment:
Not for this PR, but could skip this entire loop (or at least the distance check) if `min_cutoff is None` which might speed things up in vacuum, though probably not much.